awx/ui_next/src/components/DetailList/DetailList.jsx

```diff
@@ -11,7 +11,7 @@ const DetailList = ({ children, stacked, ...props }) => (
 export default styled(DetailList)`
   display: grid;
   grid-gap: 20px;
-  align-items: flex-start;
+  align-items: center;
```

Comment:
On a detailView when a cell requires 2 lines that label becomes off centered.  This was introduce to center the label, vertically.